roles/vendor/telegraf/templates/telegraf.toml.j2

```diff
@@ -3407,7 +3409,7 @@ metric_exclude = ["federation"]
 #   ##
 #   ## If no servers are specified, then localhost is used as the host.
 #   ## If no port is specified, 6379 is used
-#   servers = ["tcp://localhost:6379"]
+servers = ["tcp://localhost:7000", "tcp://localhost:7001"]
```

Comment:
hardcoded [here](https://github.com/ipfs-search/ipfs-search-deployment/blob/5916cd5d3c94d54d1033b76d75c52bca33d8db49/roles/vendor/redis/templates/redis_cluster.conf#L1), but we wanted a variable (list of ports)?